src/services/ui/src/pages/actions/WithdrawPackage.tsx

```diff
@@ -94,6 +94,14 @@ const WithdrawPackageForm: React.FC = ({ item }: { item?: ItemResult }) => {
             </p>
           </ActionFormIntro>
           <PackageInfo item={item} />
+          <br />
+          <h3 className="font-bold text-2xl font-sans">Attachments</h3>
+          <br />
+          <p className="font-normal">
+            Upload your supporting documentation for withdrawal or explain your
+            need for withdrawal in the <em>Additional Information section.</em>
+          </p>
+          <br />
```

Comment:
Not sure all the break lines are needed here. Breaks should be used to break up text, but these appear to be for visual layout. 

See this [article](https://www.bloomtech.com/programming-questions/what-is-br-in-html) for more.

Reply:
@kevinhaube i made the update this!